### PR TITLE
[C-2475] Add desktop favorites playlist tab

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds desktop favorites tab
Adds "create playlist" card
Adds Tile component similar to mobile
Refactors SavedPage to move albums and playlist tabs into their own files

image
image

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2475-desktop-playlist-tab

File: packages/web/src/components/tile/Tile.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Omit<ComponentProps<TileComponentType>, keyof TileOwnProps>
+
+export const Tile = <
+  T extends ElementType = ComponentType<ComponentProps<'div'>>
+>(
+  props: TileProps<T>
+) => {
+  const {
+    children,
+    size,
```
Comment (Kyle-Shanks) [inline packages/web/src/components/tile/Tile.tsx:24]: should size have a default value?

File: packages/web/src/pages/saved-page/components/desktop/AddPlaylistCard.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const AddPlaylistCard = () => {
```
Comment (Kyle-Shanks) [inline packages/web/src/pages/saved-page/components/desktop/AddPlaylistCard.tsx:1]: is this needed?

File: packages/web/src/pages/saved-page/components/desktop/PlaylistsTabPage.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      key='create_playlist'
+      size='medium'
+      as='button'
+      onClick={handleCreatePlaylist}
+      className={styles.createPlaylistCard}
+    >
+      <IconSaveFilled className={styles.createPlaylistIcon} />
+      <h4 className={styles.createPlaylistText}>{messages.createPlaylist}</h4>
+    </Tile>
+  )
```
Comment (Kyle-Shanks) [inline packages/web/src/pages/saved-page/components/desktop/PlaylistsTabPage.tsx:100]: guessing this was going to be moved to the component?

File: packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx [hunk 4]
```diff
@@ -357,9 +229,8 @@ const SavedPage = ({
           userId={account ? account.user_id : 0}
         />
       ),
-      <div className={styles.albumsWrapper} key='albums'>
-        <AlbumsTabContent />
-      </div>
+      <AlbumsTabPage key='albums' />,
+      <PlaylistsTabPage key='playlists' />
```
Comment (Kyle-Shanks) [inline packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx:233]: double checking, does this need to be behind a flag?

File: packages/web/src/components/tile/Tile.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/components/tile/Tile.tsx:24]: so i was thinking not providing it allows the content to fill however it wants for added flexibility for now.

File: packages/web/src/pages/saved-page/components/desktop/AddPlaylistCard.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/web/src/pages/saved-page/components/desktop/AddPlaylistCard.tsx:1]: no good call, just going to keep the add-playlist-card inline for now

File: packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx (hunk 4 above)
Comment (dylanjeffers) [reply packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx:233]: yeah probably wise to go through a qa run first :)

File: packages/web/src/components/tile/Tile.tsx (hunk 1 above)
Comment (Kyle-Shanks) [reply packages/web/src/components/tile/Tile.tsx:24]: cool idea, im into it

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2475-desktop-playlist-tab

**Pull request merged**